src/components/TaskBar/TaskBar.tsx

```diff
@@ -42,10 +47,12 @@ const TaskBar: React.FC<TaskBarProps> = ({
 }) => {
   return (
     <ToolBar>
-      <ToolButton style={complete ? { pointerEvents: "none" } : {}}>
+      <ToolButton disabled={complete ? true : false}>
```

Comment:
I think you can do `disabled={complete}` here.

Reply:
Updated